lib/UR/Object/Type/Initializer.pm

```diff
@@ -1087,6 +1088,20 @@ sub _normalize_property_description1 {
         }
     }
 
+    if ($new_property{default_value} && $new_property{default_from}) {
+        die qq(Can't initialize class $class_name: Property '$new_property{property_name}' has both default_value and default_from specified.);
+    }
+
+    if ($new_property{default_from}) {
+        if ($new_property{default_from} eq 1) {
+            $new_property{default_from} = '__default_' . $new_property{property_name} . '__';
+        }
+
+        unless ($class_name->can($new_property{default_from})) {
+            die qq(Can't initialize class $class_name: Property '$new_property{property_name}' has default_from specified as '$new_property{default_from}' but method does not exist.);
+        }
```

Comment:
True, but
* We support other useful things that break the module writer
* We don't actually use the module writer